subsys/bluetooth/controller/Kconfig

```diff
@@ -18,6 +18,13 @@ config BLECTRL_SLAVE_COUNT
 	  NOTE: the number of master roles is defined as
 	  BT_MAX_CONN - BLECTRL_SLAVE_COUNT
 
+config BLECTRL_MAX_CONN_EVENT_LEN
+	int "Max connection event length [us]"
+	default 7500
+	help
```

Comment:
Should have a range, minimum is 2500 us highest is 2 seconds (I think). Whatever the highest connection interval is.

Reply:
Highest connection interval is 4 seconds

Reply:
ok, added